correlation_context/HTTP_HEADER_FORMAT_RATIONALE.md

```diff
@@ -1,74 +1,72 @@
+field name. That could reduce the size of the data, particularly in http/2, where header
 compression can apply.
 
-Generally speaking `Correlation-Context` header may be split into multiple headers and 
-compression may be at the same ballpark as repeating values will be converted into a single value
-in HPAC's dynamic collection. That's said there were no profiling made to make this decision.
+Generally speaking, a `Correlation-Context` header may be split into multiple headers, and
+compression may be at the same ballpark as repeating values are converted into a single value
+in HPAC's dynamic collection. That said, there was no profiling made to make this decision.
 
 The approach with multiple headers has the following problems:
-- Name values limitation is much more pressing when the context name is used a part of a header 
+- Name values limitation is much more pressing when the context name is used a part of a header
 name.
-- The comma-separated format similar to the proposed still needs to be supported in every 
-individual header. This makes parsing harder.
-- Single header is easier to configure for tracing by many app servers.
+- The comma-separated format similar to the proposed still needs to be supported in every individual header which makes parsing harder.
+- A single header is more comfortable to configure for tracing by many app servers.
 
 ## Why not Vary-style?
 
-The [Vary](https://tools.ietf.org/html/rfc7231#section-7.1.4) approach is another alternative, 
+The [Vary](https://tools.ietf.org/html/rfc7231#section-7.1.4) approach is another alternative,
 which could be used to accomplish the same. For example, `Correlation-Context: x-b3-parentid;
-ttl=1` could tell the propagation to look at and forward the parent ID header, but only to the 
-next hop. This has an advantage of http header compression (hpack) and also weave-in with legacy 
+ttl=1` could tell the propagation to look at and forward the parent ID header, but only to the
+next hop. This has an advantage of HTTP header compression (hpack) and also weave-in with legacy
 tracing headers.
 
-Vary approach may be implemented as a new "header reference" value type `ref`. 
+Vary approach may be implemented as a new "header reference" value type `ref`.
 `Correlation-Context: x-b3-parentid;type=ref;ttl=1` if proven needed.
 
 ## Trimming of spaces
 
-Header should be human readable and editable. Thus spaces are allowed before and after the comma, equal sign, and semicolon 
-separators. It makes human-editing of headers less error-prone. It also allows better visual separation of fields when value modified manually.
+The header should be human-readable and editable. Thus spaces are allowed before and after the comma, equal sign, and semicolon separators. It makes manual editing of headers less error-prone. It also allows better visual separation of fields when value modified manually.
 
 ## Case sensitivity of names
```

Comment:
```suggestion
## Case sensitivity of keys
```

Reply:
I would consider just removing the justification for case sensitive keys entirely